tardis/adapters/sites/slurm.py

```diff
@@ -113,10 +109,12 @@ async def deploy_resource(
         self, resource_attributes: AttributeDict
     ) -> AttributeDict:
 
+        sbatch_cmdline_option_string = slurm_cmd_option_formatter(
+            self.sbatch_cmdline_options(resource_attributes.drone_uuid)
+        )
+
         request_command = (
-            "sbatch "
-            f"{self._sbatch_cmdline_option_string} "
-            f"{self._startup_command}"
+            "sbatch " f"{sbatch_cmdline_option_string} " f"{self._startup_command}"
```

Comment:
Can you please have this as a single format-string?

Reply:
Sure.